build.gradle

```diff
@@ -280,7 +280,7 @@ cargo {
 
         systemProperties {
             property("SECRETS_DIR", System.getProperty("SECRETS_DIR", file("scripts/cargo").getAbsolutePath()))
-            property("spring.profiles.active", System.getProperty("spring.profiles.active", "default"))
+            property("spring.profiles.active", System.getProperty("spring.profiles.active", "default,nurego"))
```

Comment:
Thanks for the overview in the commit message, but its best to also describe the actual changes being made to the code. I usually have something like what you have to give context, and then I have bullet points for the code changes being made - which is what we have in the guidelines - https://devcloud.swcoe.ge.com/devspace/display/RMOCM/Commit%2C+commit+messages+and+github+PR+guidelines#Commit,commitmessagesandgithubPRguidelines-Commitmessageguidelines

So something like this after what you have currently, to describe the actual changes would make the changes easier to understand:

- Update cargo configuration for local runs to use `nurego` profile in addition to `default` for running  SpringMeteringFilterIT test locally
- Update taskGraph that sets spring profile for tests to also use `nurego` profile in addition to `default` ** Is this how degraded integration tests use nurego?**
- Update nurego spring configuration files to use spring metering filter if `nurego` profile is used, and to use NoOpFilter if `nurego` profile is not used.

Reply:
Commit message is updated with additional details